scss/bitstyles/utilities/padding/padding.stories.mdx

```diff
@@ -4,48 +4,165 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
 
 # Padding
 
-Set padding on an element.
+Apply padding of a specified size and direction to an element.
+
+With default configuration, the padding classes are available in sizes `0`, `xs`, `s`, `m`, `l`, `xl`, `xxl`. This can be configured by overriding `$bitstyles-padding-sizes` e.g.
+
+```scss
+$bitstyles-padding-sizes: (
+  'positive': (
+    '0': 0,
+    'xs': spacing('xs'),
+    's': spacing('s'),
+    'm': spacing('m'),
+    'l': spacing('l'),
+    'xl': spacing('xl'),
+    'xxl': spacing('xl')
+  )
+);
+```
+
+## Padding sizes
 
 <Canvas>
+  <Story name="padding-0">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-padding-0">u-padding-0</div>
+      </div>
+    `}
+  </Story>
+  <Story name="padding-xs">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-padding-xs">u-padding-xs</div>
+      </div>
+    `}
+  </Story>
+  <Story name="padding-s">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-padding-s">u-padding-s</div>
+      </div>
+    `}
+  </Story>
   <Story name="padding-m">
     {`
       <div class="u-bg--brand-2 u-padding-m">
         <div class="u-bg--background u-padding-m">u-padding-m</div>
       </div>
     `}
   </Story>
-  <Story name="padding-0">
+  <Story name="padding-l">
     {`
-      <div class="u-bg--brand-2 u-padding-0">
-        <div class="u-bg--background u-padding-0">u-padding-0</div>
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-padding-l">u-padding-l</div>
+      </div>
+    `}
+  </Story>
+  <Story name="padding-xl">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-padding-xl">u-padding-xl</div>
+      </div>
+    `}
+  </Story>
+  <Story name="padding-xxl">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-padding-xxl">u-padding-xxl</div>
+      </div>
+    `}
+  </Story>
+</Canvas>
+
+## Directional padding
+
+Every padding size is also available directed at the single sides `top`, `right`, `bottom`, `left`, and the double directions `x` and `y`.
+
+This can be customised by overriding `$bitstyles-padding-directions` e.g.
+
+```scss
+$bitstyles-padding-directions: (
+  '': null,
+  'top': ('top'),
+  'right': ('right'),
+  'bottom': ('bottom'),
+  'left': ('left'),
+  'x': ('right', 'left'),
+  'y': ('top', 'bottom')
+);
+```
+
+The `'': null` entry is needed to render the padding classes with no directional modifiers.
```

Comment:
Same here: could this be done via a default value plus why is there no example?

Reply:
(See thread over on https://github.com/bitcrowd/bitstyles/pull/482#discussion_r642394268)